lib/actions/navigation/remove_current_page.dart

```diff
@@ -0,0 +1,30 @@
+library remove_current_page;
```

Comment:
there are just two actions now for all our navigation: PushPage and RemoveCurrentPage

Reply:
sweeeet :)